SynthDet/Assets/Scripts/Randomizers/UnifiedRotationRandomizer.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using SynthDet.RandomizerTags;
+using UnityEngine;
+using UnityEngine.Perception.Randomization.Parameters;
+using UnityEngine.Perception.Randomization.Randomizers;
+using UnityEngine.Perception.Randomization.Samplers;
+
+namespace SynthDet.Randomizers
+{
+    /// <summary>
+    /// Randomizes the rotation of objects tagged with a RotationRandomizerTag
+    /// </summary>
+    [Serializable]
+    [AddRandomizerMenu("Perception/My Unified Rotation Randomizer")]
```

Comment:
"My " should be removed from the menu item names, right?

Reply:
To make sure there are no conflicts with sample randomizers, I'll add "SynthDet" instead of "My"

Reply:
yes!